### ISSUE TITLE
DataTable: passthrough styling is ignored

### ISSUE THREAD

**Issue by dmit4git** [primefaces/primereact#5131]
### Describe the bug

Styling provided as passthrough to DataTable and Column is overridden by theme styling.

![Screencast from 10-21-2023 01_45_09 AM](https://github.com/primefaces/primereact/assets/48034657/06cf9638-a562-470c-9475-f4e36994600f)

### Reproducer

https://codesandbox.io/s/primereact-demo-forked-szt2vf

### PrimeReact version

10.0.x

### React version

18.x

### Language

ES6

### Build / Runtime

Create React App (CRA)

### Browser(s)

Chrome, Firefox

### Steps to reproduce the behavior

1. Change version of primereact dependency to `9.6.3` in package.json
2. Observe that passthrough styles are applied
3. Change it back to 10.0.x
4. See how passthrough styles are ignored

### Expected behavior

Passthrough styles should have precedence over theme styles.

**Comment by dmit4git** [comment]
This may be the same bug as #5125.

**Comment by melloware** [comment]
Yep I think it is.   I have been looking at this trying to figure it out.
